hw02-generics-collections/src/main/java/ru/otus/kovaleva/CustomerService.java

```diff
@@ -0,0 +1,37 @@
+package ru.otus.kovaleva;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    private TreeMap<Customer, String> customers;
```

Comment:
Можно добавить `final` и слева лучше объявлять тип как интерфейс:

```suggestion
    private final NavigableMap<Customer, String> customers;
```